packages/suite/src/hooks/wallet/useSendFormCompose.ts

```diff
@@ -244,9 +245,18 @@ export const useSendFormCompose = ({
     // - update context state (state.account)
     // - compose transaction with new data
     useEffect(() => {
-        if (state.account === account) return; // account didn't change
+        if (
+            !isChanged(
+                { account: state.account },
+                { account },
+                { account: ['availableBalance', 'addresses', 'balance', 'misc', 'utxo'] }, // only check relevant properties, otherwise it might recompose the transaction unnecessarily
+            )
+        ) {
+            return; // account didn't change
+        }
         if (!state.isDirty) {
             // there was no interaction with the form, just update state.account
+            console.log('returning because no interaction');
```

Comment:
we don't need to keep this log here, right?

Reply:
Right, removed as part of b5ca412c600b52811b1e59388493edcb52d1d67c